appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -45,7 +45,7 @@ export class CaseDetailsPage extends Page {
 	}
 
 	clickReviewLpaq() {
-		this.clickAccordionByButton('Documentation');
+		//this.clickAccordionByButton('Documentation');
```

Comment:
If this line of code isn't never, better to delete it rather than comment it out

Reply:
Have now removed.